frontend/src/pages/Contracts/Contract/Amendments/fields.ts

```diff
@@ -43,3 +43,11 @@ export const editFields: IEditFields[] = [
     width: "full",
   },
 ];
+
+export const initialValues = (contractId: number | undefined | string) => {
```

Comment:
The mockup shows a subcontractors field but from a quick look at the database there doesn't seem to be a table for an amendment -> subcontractor relationship. Do we know if that's a mistake in the mockup?

Reply:
I'm not really sure on, this, when i tried this out initially, it didn't even work, i do believe contract id is required.  I'm going to create some user testing tickets to see where things are working / not working.